core/tests/webdriverio_utils/LibraryPage.js

```diff
@@ -48,13 +49,20 @@ var LibraryPage = function() {
   };
   var categorySelector = forms.MultiSelectEditor(
     $('.e2e-test-search-bar-category-selector'));
+  var expSummaryTileRatingLocator = '.e2e-test-exp-summary-tile-rating';
+  var expSummaryTileObjectiveLocator = '.e2e-test-exp-summary-tile-objective';
   var languageSelector = forms.MultiSelectEditor(
     $('.e2e-test-search-bar-language-selector'));
 
   // Returns a promise of all explorations with the given name.
-  var _getExplorationElements = async function(name) {
-    var allExplorationSummaryTile = allExplorationSummaryTileSelector();
-    return await allExplorationSummaryTile.filter(
+  var _getExplorationElements = async function(name, isHidden = false) {
```

Comment:
Can you please add a comment above explaining the significance of `isHidden`?

Reply:
Done.